atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/ConflictDetectionManager.java

```diff
@@ -39,7 +39,11 @@ public class ConflictDetectionManager {
      *  (This has always been the behavior of this class; I'm simply calling it out)
      */
     public ConflictDetectionManager(CacheLoader<TableReference, ConflictHandler> loader) {
-        this.cache = Caffeine.newBuilder().maximumSize(100_000).build(loader);
+        this.cache = Caffeine.newBuilder()
```

Comment:
Seems like the tests expects the loader to even load null values while async cache does not allow that, because of which the test is failing.

```
Throws:
NullPointerException – if the specified key is null or if the future returned by the AsyncCacheLoader is null
```

Having null ConflictHandler is an expected state ?